packages/docusaurus-theme-classic/src/theme/Heading/index.tsx

```diff
@@ -38,6 +38,7 @@ export default function Heading({as: As, id, ...props}: Props): JSX.Element {
       <Link
         className="hash-link"
         to={`#${id}`}
+        aria-labelledby={id}
```

Comment:
wouldn't it be better to use `children` instead of `id` there? 

(ie "Project Structure" instead of "project-structure")

Ids are not necessarily as human readable as the original text of the heading

Reply:
`aria-labelledby` accepts the id of an element containing some text. And because the heading contains e.g. "Project structure" text, it becomes the accessible name of the link as well.

But if you are mentioning `children`, it would be even better to do this: `aria-label={children}`. I just was not sure about TypeScript here (not an expert). Will children always be only text content or it's possible there will be some tags too?

Reply:
hmmm right it might not always be a string